src/gba/hardware.c

```diff
@@ -293,7 +293,11 @@ void _rtcUpdateClock(struct GBACartridgeHardware* hw) {
 		t = time(0);
 	}
 	struct tm date;
+#ifdef __CELLOS_LV2__
```

Comment:
Please undefine `HAVE_LOCALTIME_R` in the makefile and add an implementation at the bottom of `src/util/formatting.c` -- you'll see there are already a small smattering of them there to give you an idea.